compiler/damlc/lib/DA/Cli/Damlc/Packaging.hs

```diff
@@ -323,6 +297,22 @@ settings =
   , ("cross compiling", "NO")
   ]
 
+-- Register a dar dependency in the package database
+registerDepInPkgDb :: FilePath -> FilePath -> IO ()
+registerDepInPkgDb dbPath depPath = do
+    linkDirRec (configDir depPath) (dbPath </> "package.conf.d")
+    linkDirRec (sourcesDir depPath) dbPath
+    linkDirRec (mainDir depPath) dbPath
+    recachePkgDb dbPath
+
+linkDirRec :: FilePath -> FilePath -> IO ()
+linkDirRec from to = do
+    srcs <- listFilesRecursive from
```

Comment:
why symlink individual files instead of symlinking the directory? Do we merge multiple directories?

Reply:
changed this back to copying files, so there it makes sense again.